SpatialGDK/Extras/schema/global_state_manager.schema

```diff
@@ -18,6 +18,11 @@ component DeploymentMap {
     uint32 schema_hash = 4;
 }
 
+component SnapshotVersion {
+    id = 9990;
+    uint32 version_number = 1;
```

Comment:
```suggestion
    uint64 version_number = 1;
```
As I think the rest of the code treats it that way?

Reply:
That's a lot of versions

Reply:
It is version and a schema hash.